### ISSUE TITLE
Loss of Dimension Identification in X3D file

### ISSUE THREAD

**Issue by Emmanuelhfc** [usnistgov/SFA#6]
Hello,

I use SFA to convert .step files to X3D format. In previous versions, the X3D files generated included identification in the ID attribute of tags indicating which dimension they represented. This feature was particularly useful for various post-processing tasks, such as hiding specific dimensions, changing their colors, and more. However, in the latest version of the repository, this identification is no longer present.

#### Example
Here is a simplified example to illustrate the issue:

**Version 5.12**
```xml
<Shape id='RD21@Annotations@10011907666SWA000.moAssembly_c'>
  <Appearance>
    <Material diffuseColor='0 0 0' emissiveColor='0 0 0' shininess='0'/>
  </Appearance>
  <IndexedFaceSet solid='false' coordIndex='2 0 1 -1'>
    <Coordinate DEF='coord84' point='555.492 -510.0 0.295 548.742 -483.0 0.274 562.242 -483.0 0.274'/>
  </IndexedFaceSet>
</Shape>
```

**Version 5.14**
```xml
<Shape>
  <Appearance>
    <Material diffuseColor='0 0 0' emissiveColor='0 0 0' shininess='0'/>
  </Appearance>
  <IndexedFaceSet solid='false' coordIndex='2 0 1 -1'>
    <Coordinate DEF='coord84' point='555.492 -510.0 0.295 548.742 -483.0 0.274 562.242 -483.0 0.274'/>
  </IndexedFaceSet>
</Shape>
````

Thank you for your attention to this matter.

**Comment by robert-lipman** [comment]
Sorry that the change in SFA 5.14 affected your use of the X3D files.  There is a reason that the Shape IDs were removed, but I see that it was too broad.  I will restore them in the next version.  You can revert back to 5.12 to have the Shape IDs.

**Comment by Emmanuelhfc** [comment]
Perfect, thanks for your attention!

**Comment by robert-lipman** [comment]
This issue is fixed in version 5.16.
